connector/signaltometricsconnector/internal/customottl/adjustedcount.go

```diff
@@ -0,0 +1,41 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package customottl // import "github.com/open-telemetry/opentelemetry-collector-contrib/connector/signaltometricsconnector/internal/customottl"
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/contexts/ottlspan"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/sampling"
+)
+
+func NewAdjustedCountFactory() ottl.Factory[ottlspan.TransformContext] {
```

Comment:
[**For reviewers**] I am hoping this will get added to the ottl package and then we can remove it from here. I have an issue open for this: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36572